includes/wikia/models/NavigationModel.class.php

```diff
@@ -1,16 +1,17 @@
 <?php
+
 /**
  * Model for Wiki Navigation
  *
  * @author Jakub "Student" Olek
  */
-
 class NavigationModel extends WikiaModel {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "NavigationModel" has 21 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1448)